src/main/webapp/edit-recipe.html

```diff
@@ -45,6 +46,20 @@ <h2>Steps</h2>
       <textarea id="step2" name="step2" rows="1"></textarea></br></br>
       <label for="step3">Step 3</label></br>
       <textarea id="step3" name="step3" rows="1"></textarea></br></br>
+=======
+    <div id="Tags">
+      <parameter-input name="Tag" index="0" id="Tag0"></parameter-input>
+    </div>
+
+    <h2>Ingredients</h2>
+    <div id="Ingredients">
+      <parameter-input name="Ingredient" index="0" id="Ingredient0"></parameter-input>
+    </div>
+
+    <h2>Steps</h2>
+    <div id="Steps">
+      <parameter-input name="Step" index="0" id="Step0"></parameter-input>
+>>>>>>> ae60ec3... add support for populating spin-offs with any number of parameters
```

Comment:
Please remove the merge conflict indicators;

Reply:
I'm embarrased smh